docs/developer-manual/terms-of-service.md

```diff
@@ -0,0 +1,157 @@
+against the Protected Parties by other members of the organization or entity on
+whose behalf you may be using the Website or Services; or your infringement or
+misappropriation of the rights of any other person or entity. If you are
+obligated to indemnify any Protected Party, the Oracle (or, at its discretion,
+the applicable Protected Party) will have the right, in its sole discretion, to
+control any action or proceeding and to determine whether the Oracle wishes to
+settle, and if so, on what terms.
+
+## DISCLOSURES; DISCLAIMERS
+
+To the maximum extent permitted under Applicable Law, the Website and the
+Services (and any of their content or functionality) provided by or on behalf of
+us are provided on an “AS IS” and “AS AVAILABLE” basis, and we expressly
+disclaim, and you hereby waive, any representations, conditions or warranties of
+any kind, whether express or implied, legal, statutory or otherwise, or arising
+from statute, otherwise in law, course of dealing, or usage of trade, including,
+without limitation, the implied or legal warranties and conditions of
+merchantability, merchantable quality, quality or fitness for a particular
+purpose, title, security, availability, reliability, accuracy, quiet enjoyment
+and non-infringement of third party rights. Without limiting the foregoing, we
+do not represent or warrant that the Website or the Services
+(including any related data) will be uninterrupted, available at any particular
+time or error-free. Further, we do not warrant that errors in the Website or the
+Service are correctable or will be corrected.
+
+## EXCLUSION OF CONSEQUENTIAL AND RELATED DAMAGES
+
+In no event will the Oracle, together with any Protected Party, be liable for
+any incidental, indirect, special, punitive, exemplary, consequential or similar
+damages or liabilities whatsoever (including, without limitation, damages for
+loss of data, information, revenue, goodwill, profits or other business or
+financial benefit) arising out of or in connection with the Website, the
+Services and the Orcfax Network (and any of their content and functionality),
+any execution or settlement of a transaction, any performance or non-performance
+of the Services, your Distributed Ledger Technology assets, other digital
+assets, Fact Tokens or any other product, service or other item provided by or
+on behalf of a Protected Party, whether under contract, tort
+(including negligence), civil liability, statute, strict liability, breach of
+warranties, or under any other theory of liability, and whether or not any
+Protected Party has been advised of, knew of or should have known of the
+possibility of such damages and notwithstanding any failure of the essential
+purpose of these Terms or any limited remedy nor is the Oracle in any way
+responsible for the execution or settlement of transactions between users of
+Orcfax software or the Orcfax Network.
+
+## LIMITATION OF LIABILITY
```

Comment:
I guess this section is: "if you do sue us, the TOS makes it clear to you that you can have your money back (fees totaling no more than 12 months)?

^^ above my pay-grade - but does this mean we should escrow fees from parties for 12 months before use?

nb. also, do we need to specify something like, in the currency in which the fee was paid? e.g. if paid in ADA you get back equivalent ADA?

Reply:
> TOS makes it clear to you that you can have your money back (fees totaling no more than 12 months)?

If someone were to sue, and was successful, the extent to which they could sue is limited to 12 mo worth of publication costs (is my understanding).

As for escrow... This is above my pay grade but I imagine Orcfax will need to have funds available to cover these kinds of situations. Otherwise it could be catastrophic for the company



> also, do we need to specify something like, in the currency in which the fee was paid? e.g. if paid in ADA you get back equivalent ADA?

I imagine so. This is where a professional would be useful.